locale/en.json

```diff
@@ -131,10 +131,11 @@
   "NEW_INDIVIDUAL_CREATED_DESCRIPTION": "New individual created.",
   "PENDING_BULK_IMPORT_MESSAGE": "You reported {sightingCount} sightings on {date}. You must finish processing this bulk import before starting another.",
   "HISTORY": "History",
+  "IMAGE_PROCESSING_SKIPPED_MESSAGE": "Image processing skipped because no images were submitted.",
   "DETECTION_SKIPPED_MESSAGE": "Detection skipped, either there are no images or no detection model was specified.",
   "CURATION_SKIPPED_MESSAGE": "Curation skipped - no images to curate.",
-  "MATCHING_SKIPPED_MESSAGE": "Matching skipped - no images to match.",
-  "MATCHING": "Matching",
+  "IDENTIFICATION_SKIPPED_MESSAGE": "Identification skipped - no images to match.",
```

Comment:
there _might_ be other reasons id was skipped, so this message ("no images to match") might be confusing.
isnt there an option via the ui that the user can pick to skip id?

Reply:
Yes. These messages need to be updated, but for this PR, I wanted to leave everything as it was except how the stages were calculated. This is just a change from "Matching" to "Identification" in the string.

Reply:
I agree with Jon, but also recognize that this is something to improve on later.  I'm OK with this placeholder message for now.